Gems/Atom/RHI/Code/Source/RHI/RayTracingAccelerationStructure.cpp

```diff
@@ -26,13 +26,13 @@ namespace AZ::RHI
                 ->IndexBuffer(geometry.m_indexBuffer.GetDeviceIndexBufferView(deviceIndex));
         }
 
-        descriptor.BuildFlags(m_buildFlags);
-
         if(m_aabb.has_value())
```

Comment:
missing space between if and (

Reply:
I dont think I am supposed to change just the formatting of unchanged codelines.